.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+
+python:
+  - "3.6"
+sudo: required
+
+install:
+  - pip install black
+  -
+
+script:
+  - black pymc4/ --check
```

Comment:
We should set the `--py36` flag.

Also, did we agree on line lengths of 100 or something? I'm not sure.

Reply:
I don't think we explicitly agreed. In the other PR I set 100 because that's what we did in ArviZ and it got merged so ¯\_(ツ)_/¯

Good idea on py36 flag

Reply:
I'll do the 3.6 flag in another PR since it will change the formatting